peps/pep-0774.rst

```diff
@@ -0,0 +1,250 @@
+PEP: 774
+Title: Removing the LLVM requirement for JIT builds
+Author: Savannah Ostrowski <savannah@python.org>
```

Comment:
I didn't add `Discussions-To` quite yet since I wanted to 1) have the PEP number confirmed and 2) have this merged. If there's an order of operations that is standard here, please let me know.

Reply:
Yeah. I think that's pretty standard (updating with a link to the thread right after).

Reply:
Yep, this is the usual way 👍 

The other way is to, _just before_ merge, open the thread then immediately update the PR and merge.

(The not recommended way is to open the thread with the RTD preview at the same time the PR is open, as that thread can live a long time and the preview link will eventually 404.)

And confirming 774 for this PEP (it has nice symmetry with [744](https://peps.python.org/pep-0744/) :)